px-data-grid-paginated.html

```diff
@@ -172,50 +230,311 @@
+             *     in the column. Used in the advanced filter UI to show
+             *     different filter options for each data type. Does not change
+             *     the renderer or impact sorting.
+             *
+             *   * `{string} renderer` - The name of the web component used to
+             *     render each cell in the column. There are three built-in
+             *     renderers available: `'px-data-grid-string-renderer'` (the
+             *     default renderer), `'px-data-grid-number-renderer'`, and
+             *     `'px-data-grid-date-renderer'`. You can also create a custom
+             *     renderer and use its name here.
+             *
+             *   * `{object} rendererConfig` - Settings to pass to the renderer
+             *     that change how data is displayed to the user. See the
+             *     examples below for information on which options are available
+             *     for each built-in renderer.
+             *
+             * The following options are available for columns of type `date`:
+             *
+             *   * `{array} dateRanges` - List of pre-defined date ranges that
+             *     will appear in advanced filter dropdown for this column.
+             *     See the examples below for examples on how to format
+             *     each date range.
+             *
+             * The following options are available for columns of type `number`:
+             *
+             *   * `{number} minBound` - Defines a minimum bound of a number in
+             *     the advanced filter slider for this column. If both minBound
+             *     and maxBound properties are defined, the advanced filter will
+             *     display slider instead of dropdown.
+             *
+             *   * `{number} maxBound` - Defines a maximum bound of a number in
+             *     the advanced filter slider for this column.
+             *
+             *   * `{boolean} hidden=false` - Hides the column from the user
+             *     when the grid is rendered. Automatically updated when the
+             *     user shows/hides columns using action menu(s).
+             *
+             *   * `{boolean} frozen=false` - Freezes the column. When the user
+             *     scrolls the grid horizontally frozen columns will not move.
+             *     Automatically updated when the user freezes/unfreezes columns
+             *     using action menu(s).
+             *
+             *   * `{boolean} required=false` - Used in renderers to during
+             *     editing. If the column is required, the user will not be
+             *     allowed to enter a blank value.
+             *
+             *   * `{number} flexGrow=1` - Sets the relative size of the column
+             *     compared to other columns in the grid. Equivalent to the CSS
+             *     `flex-grow` property.
+             *
+             * ## `column.rendererConfig` options
```

Comment:
The layout seems a bit confusing here, you described the `rendererConfig` property, then included some properties relevant to each column type, but now you're diving into the various `rendererConfig` options - maybe rearrange so that the `rendererConfig` property immediately precedes this section. Or group all the date/number stuff together?